articles/meta-learning/index.md

```diff
@@ -0,0 +1,101 @@
+
+### Meta-Learning
+
+Very simply defined, meta-learning means learning to learn. It is a learning process that applies to understand algorithms to metadata. Metadata is data that describes other data. Traditional machine learning has us use a sizeable dataset exclusive to a given task to train a model. This is a very involving process. It contrasts with how humans take in new information and learn new skills. Human beings do not need a large pool of examples to know. We learn very quickly and efficiently from a handful of examples. Taking inspiration from how human beings learn, meta-learning attempts to automate traditional machine learning challenges. It seeks to apply machine learning to learn the most suitable parameters and algorithms for a given task.
+
+A hypothesis space may be defined as a set of all hypotheses that may be returned by a machine learning model. Meta-learning impacts the hypothesis space for learning algorithms. This might be through the tuning of hyperparameters or the selection of features. It may also change an algorithm's learning rules by altering how the algorithm searches the hypothesis space. We shall better understand meta-learning in the latter sections of this article by exploring this technique's approaches.
+
+### Steps
+
+**Inclusion of a Learning Algorithm.** A learning algorithm is critical in the process of "learning how to learn". The learner is used to learn the optimal parameters as well as algorithms for a given task. To carry out meta-learning, we need such a learning algorithm.
+
+**Dynamic Inductive Bias.** Inductive bias is the set of assumptions a learning algorithm uses to make predictions. This is when the algorithm is given inputs it has never come across. Dynamic bias induction refers to where bias is constructed as a function of the learning task. This simply means that the inductive bias of a learner is altered to match a given task. Essential aspects of the learner can be changed to achieve a dynamic inductive bias. These aspects include the representation of the hypothesis or parameters.
+
+**Extracting insights from Metadata.** Metadata is a set of data that describes other data. We could call it data about data. In the context of meta-learning, it is made up of knowledge of prior learning experiences. This helps in the development of a hypothesis for a new task.
+
+### Approaches to Meta-Learning Algorithms
+
+Various literature details different kinds of approaches to meta-learning. We cover three approaches in this section.
+
+#### Optimized Meta-Learning
+
+Wikipedia defines a [hyperparameter](https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning)) as a parameter whose value is used to control the learning process. It is a parameter that is defined before the start of the learning process. Hyperparameters have a direct impact on the quality of the training process. Hyperparameters can be tuned. An example of a hyperparameter is the number of branches in a decision tree.
```

Comment:
Need not say Wikipedia defines hyperparameter. The definition of hyperparameter is a standard one.